example/game/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,32 @@
+<resources>
+    <string name="app_name">Game</string>
+
+    <string name="start_game_description">Please select one option</string>
+    <string name="start_game">Start Game</string>
+    <string name="join_game">Join Game</string>
+    <string name="play">Play Game</string>
+
+    <string name="client">Client Side</string>
+    <string name="looking_for_clients">Looking for clients…</string>
+
+    <string name="server">Server Side</string>
+    <string name="scanning_for_server">Scanning for Server…</string>
+
+    <string name="connected">Connected</string>
+    <string name="loading">Loading…</string>
+    <string name="disconnected">Disconnected</string>
+    <string name="initializing">Waiting for server to connect…</string>
+    <string name="downloading">Downloading…</string>
+
+    <string name="next">Next</string>
+    <string name="done">Done</string>
+
+    <string name="game_over">Game Over!!</string>
+    <string name="show_result">Show Result</string>
+    <string name="show_result_des">Click on \'Show Result\' to show result</string>
+
+    <string name="ask_players_name">Enter player\'s name</string>
+    <string name="send_players_name">Send</string>
+    <string name="duplicate_name_error">Duplicate Name!!</string>
```

Comment:
Perhaps "Another user has already taken that name, please select a different one."?